spring-cloud-gcp-data-firestore/src/main/java/org/springframework/cloud/gcp/data/firestore/repository/query/PartTreeFirestoreQuery.java

```diff
@@ -78,6 +80,7 @@ public PartTreeFirestoreQuery(FirestoreQueryMethod queryMethod, FirestoreReactiv
 		ReturnedType returnedType = queryMethod.getResultProcessor().getReturnedType();
 		this.tree = new PartTree(queryMethod.getName(), returnedType.getDomainType());
 		this.persistentEntity = mappingContext.getPersistentEntity(returnedType.getDomainType());
+		this.classMapper = mappingContext.getClassMapper();
```

Comment:
Again, maybe just accept a `classMapper` in the constructor directly rather than going through this `mappingContext`.

Reply:
Keeping them separate is fine.